From: Dominik George <nik@naturalnet.de>
Subject: Use absolute path to PulseAudio modules
 The modules are installed to their locations by the
 xrdp-build-pulse-modules script from the former
 xrdp-pulseaudio-installer package. The upstream
 pulseaudio-module-xrdp/wiki/README instructions
 install the module into the PA directory and do
 not need this patch.

--- a/instfiles/pulse/default.pa
+++ b/instfiles/pulse/default.pa
@@ -8,5 +8,11 @@ load-module module-xrdp-sink
 .ifexists module-xrdp-source.so
 load-module module-xrdp-source
 .endif
+.ifexists /var/lib/xrdp-pulseaudio-installer/module-xrdp-sink.so
+load-module /var/lib/xrdp-pulseaudio-installer/module-xrdp-sink.so
+.endif
+.ifexists /var/lib/xrdp-pulseaudio-installer/module-xrdp-source.so
+load-module /var/lib/xrdp-pulseaudio-installer/module-xrdp-source.so
+.endif
 load-module module-native-protocol-unix
 
